constants/members.json

```diff
@@ -164,9 +164,9 @@
     {
       "name": "Eugene Zhang",
       "role": "developer",
-      "term": 1235,
-      "teams": ["casa"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fw23headshots%2Fdev_eugene_zhang.jpeg?alt=media&token=8b3bf4cc-95f0-4c16-8ac0-2c05bea159b3"
+      "term": 1236,
+      "teams": ["cas"],
```

Comment:
this is for childrens aid society right? The code still seems to be "casa" and not "cas", correct me if im wrong though

Reply:
@Jess2772 is correct. @LucyWuu my apologies, I should've been more specific in relating the team name with the one in the code

Reply:
@HeetShah no worries! I should've confirmed it with you. @Jess2772 thanks for pointing this out!